src/runtime/virtcontainers/clh.go

```diff
@@ -1385,7 +1401,8 @@ func (clh *cloudHypervisor) isClhRunning(timeout uint) (bool, error) {
 
 	pid := clh.state.PID
 
-	if err := syscall.Kill(pid, syscall.Signal(0)); err != nil {
+	err := syscall.Kill(pid, syscall.Signal(0))
+	if err != nil {
```

Comment:
Nit: this should go into the second commit.

Reply:
Fixed.